velox/common/testutil/TestValue.h

```diff
@@ -45,19 +45,19 @@ class TestValue {
   template <typename T>
   static void set(
       const std::string& injectionPoint,
-      std::function<void(const T*)> injectionCb);
+      std::function<void(T*)> injectionCb);
 
   /// Invoked by the test code to unregister a callback hook at the specified
   /// execution point.
   static void clear(const std::string& injectionPoint);
 
   /// Invoked by the production code to try to invoke the test callback hook
   /// with 'testData' if there is one registered at the specified execution
-  /// point. 'testData' capture the production execution state in readonly mode.
-  static void notify(const std::string& injectionPoint, const void* testData);
+  /// point. 'testData' capture the mutable production execution state.
+  static void adjust(const std::string& injectionPoint, void* testData);
```

Comment:
It would be nice to mention this change in the PR description.

Reply:
It has already been included :)